docs/overloading-wp-behavior.md

```diff
@@ -148,6 +148,133 @@ export default function Page() {
 
 ---
 
+## Overloading React Parser Options
+
+SnapWP allows you to pass custom options to [**react-parser**](../packages/next/src/react-parser/index.tsx) . This is useful when you want to overload default options of react-parser.
```

Comment:
There should be some mention and link to `html-react-parser`, "React Parser Options" isnt known terminology.

Reply:
@BhumikP bumping